src/views/admin/PersonManagementView.vue

```diff
@@ -35,15 +69,37 @@
     <ResultTable
       data-testid="person-table"
       :header="$t('admin.person.management')"
-      :items="personStore.allPersons || []"
+      :items="personenWithUebersicht || []"
       :loading="personStore.loading"
       :headers="headers"
       @onHandleRowClick="navigateToPersonDetails"
       @onUpdateTable="personStore.getAllPersons()"
       :totalItems="personStore.totalPersons"
       item-value-path="person.id"
-    ></ResultTable>
+      ><template v-slot:[`item.rolle`]="{ item }">
+        <div
+          class="ellipsis-wrapper"
+          :title="item.rolle"
+        >
+          {{ item.rolle }}
+        </div> </template
+      ><template v-slot:[`item.administrationsebenen`]="{ item }">
+        <div
+          class="ellipsis-wrapper"
+          :title="item.administrationsebenen"
+        >
+          {{ item.administrationsebenen }}
+        </div>
+      </template></ResultTable
+    >
   </div>
 </template>
 
-<style></style>
+<style>
+  .ellipsis-wrapper {
```

Comment:
Bitte global in main.scss definieren, dann können wir die Klasse überall benutzen (z.B. auch bei der Seite eines Benutzers, für den Schulnamen)

Reply:
Done.